docs/proposals/scheduling/20240206-multi-tree-quota.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Multi Tree Elastic Quota
+authors:
+  - "@shaloulcy"
+reviewers:
+  - "@hormes"
+  - "@buptcozy"
+  - "@eahydra"
+creation-date: 2024-02-06
+last-updated: 2024-02-06
+status: provisional
+---
+
+# Multi Tree Elastic Quota
+
+## Table of Contents
+
+- [Multi Tree Elastic Quota](#multi-tree-elastic-quota)
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+      - [Goals](#goals)
+      - [Non-Goals/Future Work](#non-goalsfuture-work)
+  - [Proposal](#proposal)
+    	- [User Stories](#user-stories)
+      		- [Story 1](#story-1)
+      		- [Story 2](#story-2)
+    	- [Architecture](#architecture)
+    	- [Implementation Details](#implementation-details)
+    	  - [API](#api)
+    	  - [Koord-manager](#koord-manager)
+    	  - [Koord-scheduler](#koord-scheduler)
+  - [Implementation History](#implementation-history)
+
+
+## Summary
+
+
+The proposal provides a mechanism to construct multiple trees for elastic quota. Different quota tree can include different nodes. It will help the cluster administrators to manage the node resource more granularly
+
+
+## Motivation
+
+[Elastic Quota](https://github.com/koordinator-sh/koordinator/blob/main/docs/proposals/scheduling/20220722-multi-hierarchy-elastic-quota-management.md) provides a balance between enforcing limits to prevent resource exhaustion and allowing for efficient use of resources. Currently, there is only a quota tree in cluster. The root quota is `koordinator-root-quota`, all other quotas is the descendants of the root quota. In addition, all nodes resources belong to the only quota tree. 
+
+In certain scenarios, we need devide nodes resources into different groups and put certain workloads in particular group. For example, we can devide nodes by availability zones and put some workloads in particular availability zone. But the elastic quota tree includes all nodes, it's not accurate. 
```

Comment:
It can be mentioned here that k8s products from cloud providers usually provide users with the concept of node pools to support users in managing different workloads.

Reply:
fix it